src/App.js

```diff
@@ -1,20 +1,24 @@
 // App.js
-/* eslint-disable */
 
+import React from 'react';
 import './App.css';
-import React, { Component, useState, useEffect } from 'react';
-import Calc from './components/Calculator';
+import { Route, Routes } from 'react-router-dom';
+import WebHeader from './components/wHeader';
+import WebHome from './components/wHome';
+import WebCalc from './components/wCalc';
+import WebQuote from './components/wQuote';
 
-export default class App extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
-
-  render() {
-    return (
-      <Calc />
-    );
-  }
+function App() {
+  return (
+    <div className="container">
+      <WebHeader />
+      <Routes>
+        <Route exact path="/mathmg" element={<WebHome />} />
```

Comment:
- [ ] **[OPTIONAL]** When I opened your app the root path shows an empty page, this is because you do not have a root path. I suggest you make `element={<WebHome />}` your root path so that once your app opens on the browser the root path displays the `<WebHome />` component. Also go to the `<WebHeader />` and replace line `12` with `<li><a href="/">Home</a></li>`.

```suggestion
      <Routes>
        <Route exact path="/" element={<WebHome />} />
```